topo/dva/forch.yaml

```diff
@@ -5,8 +5,11 @@ site:
     nz-kiwi-ctl1:
       fqdn: localhost.n
       port: 9020
-static_device_info:
+orchestration:
+    run_faucetizer: True
```

Comment:
this is where I'm thinking the run_faucetizer variable is redundant.... if "dynamic_config_file" is defined, then that's all we need to know!